main/app.py

```diff
@@ -0,0 +1,336 @@
+    """Description for amity"""
+
+    def __init__(self):
+        # variable used in create_rooms
+        self.rooms = {
+            'O': [],
+            'L': []
+        }
+        # variable used in add_person
+        self.people_data = {'Staff': [],
+                            'Fellow': []
+                            }
+        self.allocate = {
+            'O': {' ': []},
+            'L': {' ': []}
+        }
+
+        self.room_type = "O" or "L"
+        self.name = ""
+        name = self.name
+        self.personnel_type = ""
+        personnel_type = self.personnel_type
+        self.want_accommodation = ""
+        want_accommodation = self.want_accommodation
+
+        self.offices = []
+        self.full_name = []
+        self.allocated_room = []
+
+        self.person_name = ""
+        self.person_type = ""
+        self.room_name = ""
+        self.people_in_room = []
+
+        self.person_name = []
+        self.person_type = ""
+
+        self.offices = []
+        self.housing = []
+
+        self.available_office = []
+        self.available_housing = []
+
+        self.office_allocations = []
+        self.living_alllocations = []
+        self.office_occupants = []
+        self.unallocated_offices = []
+        self.living_occupants = []
+        self.unallocated_houses = []
+        self.office = []
+        self.housing = []
+
+        self.living_occupied = 0
+        self.office_occupied = 0
+        file = self.file = " "
+        allocated = self.allocated = []
+
+        self.conn = sqlite3.connect("amity.sqlite")
+        self.conn.row_factory = sqlite3.Row
+        self.connect = self.conn.cursor()
+        self.connect.execute(
+            "CREATE TABLE IF NOT EXISTS Rooms(id INTEGER PRIMARY KEY AUTOINCREMENT, Name TEXT, type TEXT)")
+
+        self.connect.execute(
+            "CREATE TABLE IF NOT EXISTS Persons(id INTEGER PRIMARY KEY AUTOINCREMENT, Name TEXT, Personnel_type TEXT, want_accommodation TEXT, office_accommodation TEXT , living_accomodation TEXT)")
+
+    def create_rooms(self, room):
+        """allows user to enter a list of room names specifying
+                whether office or living spaces"""
+        room_type = raw_input(
+            "Enter room type: \n O: Office space \n L: Living space: \n")
+
+        while room_type != "O" and room_type != "L":
+            room_type = raw_input(
+                "Try again. Enter Room Type:\n O: Office space \n L: Living space: \n")
+        for room_name in room:
+            if room_type.upper() == 'O':
+                self.rooms[room_type.upper()].append(room_name)
+            else:
+                self.rooms['L'].append(room_name)
+            self.connect.execute(
+                "INSERT INTO Rooms (Name, type) VALUES (?, ?)", [room_name, room_type])
+            self.conn.commit()
+        print('New rooms succesfully created')
+
+    def office_space_count(self, off_name):
+        ''' keep track of offices allocated'''
```

Comment:
**Code Quality: Nitpick**

You should use proper capitalization for Doc string comments like these.